### PR TITLE
Added ability to disable OpenSearch audit logs

### PR DESCRIPTION
**Pull request by seriva**
- Added ability to disable OpenSearch audit logs
- Fixed some validation for logging

### CODE REVIEW

**Review by cicharka — DISMISSED** [review]

File: schema/common/defaults/configuration/opensearch.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ specification:
     - readall
     - snapshotrestore
     # - logstash
-    # - kibanaserver
+    # - kibanaservers
```
Comment (to-bar) [inline schema/common/defaults/configuration/opensearch.yml]: Changed by purpose? 🤔 I see `kibanauser` in https://opensearch.org/docs/latest/security-plugin/configuration/yaml#internal_usersyml

File: ansible/playbooks/roles/opensearch/templates/opensearch.yml.j2 [hunk 2]
```diff
@@ -126,7 +126,10 @@ plugins.security.nodes_dn:
   {% endfor %}
 {% endif %}
 
-plugins.security.audit.type: internal_opensearch
+{% if specification.opensearch_security.audit.type is defined and specification.opensearch_security.audit.type|length %}
+plugins.security.audit.type: {{ specification.opensearch_security.audit.type }}
+{% endif %}
+
```
Comment (to-bar) [inline ansible/playbooks/roles/opensearch/templates/opensearch.yml.j2]:
```suggestion
```

File: schema/common/defaults/configuration/opensearch.yml (hunk 1 above)
Comment (seriva) [reply schema/common/defaults/configuration/opensearch.yml]: Probably added by accident.

**Review by to-bar — APPROVED** [review]

**Review by cicharka — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**